packages/fluent-theme/src/fluent/styles/CommandBarButton.styles.ts

```diff
@@ -6,10 +6,92 @@ export const CommandBarButtonStyles = (props: IButtonProps): Partial<IButtonStyl
   if (!theme) {
     throw new Error('Theme is undefined or null.');
   }
+  const { palette, semanticColors } = theme;
 
   return {
-    root: {
-      ...getFocusStyle(theme, 2)
+    root: [
+      { ...getFocusStyle(theme, 2) },
+      {
+        backgroundColor: palette.white
+      }
+    ],
+
+    rootHovered: {
+      backgroundColor: palette.neutralLighter,
+      selectors: {
+        '.ms-Button-icon': {
+          color: palette.themeDarkAlt
+        }
+      }
+    },
+
+    rootPressed: {
+      backgroundColor: palette.neutralLight,
+      color: palette.neutralDark,
+      selectors: {
+        '.ms-Button-icon': {
+          color: palette.themeDark
+        }
+      }
+    },
+
+    rootChecked: {
+      backgroundColor: palette.neutralLight,
+      color: palette.neutralDark,
+      selectors: {
+        '.ms-Button-icon': {
+          color: palette.themeDark
+        }
+      }
+    },
+
+    rootExpanded: {
+      color: palette.neutralDark,
+      selectors: {
+        '.ms-Button-icon': {
+          color: palette.themeDark
+        }
+      }
+    },
+
+    rootDisabled: {
+      backgroundColor: palette.white,
+      selectors: {
+        '.ms-Button-icon': {
+          color: semanticColors.disabledBodySubtext
+        }
+      }
+    },
+
+    splitButtonMenuButton: {
```

Comment:
Interesting... I now wonder if I made the appropriate style changes for a split button in a command bar in `fabric-7` branch. Thanks for making the fixes.